packages/grafana-ui/src/components/ResourceCard/ResourceCard.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+export interface Props {
+  resourceName: string;
+  imageUrl?: string;
+  description?: string;
+  url?: string;
+  type?: string;
+  isDefault?: boolean;
+  actions?: ReactNode;
```

Comment:
wish you continued work in small steps and asked for feedback on https://github.com/grafana/grafana/pull/21352/files instead, as this is not what I defined in https://github.com/grafana/grafana/pull/21352/files#diff-486446653e4a41794ee9043e371eb88cR3 

The thinking was a more generic component, that could have any info items, not just url & type, & isDefault.

Reply:
Oops, missed this PR somehow. On it.